upgrades/2025_03_31_140423_add_menu_frame_whitelist.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use ProcessMaker\Models\Setting;
```

Comment:
add use use ProcessMaker\Models\SettingsMenus; for avoid errors

Reply:
done